src/components/playlist/video/videoPlayer.tsx

```
@@ -0,0 +1,52 @@
+
+type VideoPlayerProps = {
+    currentVideo: Video | undefined;
+    thumbnailUrl: string;
+    handleNextVideo: (index?: number, isAutoPlay?: boolean) => void;
+}
+
+const VideoPlayer = ({ currentVideo, thumbnailUrl, handleNextVideo }: VideoPlayerProps) => {
+
+    if (!currentVideo) {
+        return (
+            <section className="flex flex-col items-start justify-center w-full mt-3 mb-8">
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)